### ISSUE TITLE
multiple 1click-hpc clusters with the same FSx will crash enginframe

### ISSUE THREAD

**Issue by rvencu** [aws-samples/1click-hpc#27]
Since the /fsx/nice location is not unique to the cluster, starting multiple clusters with the same fsx will overwrite the portal data for older clusters

**Comment by nicolaven** [comment]
yes, right. we could consider using something like /fsx/nice/{instanceID}/ instead.
what do you think ?

**Comment by rvencu** [comment]
or what about cluster name that has to be unique anyway?

`"export CLUSTER_NAME=${AWS::StackName}"`

**Comment by nicolaven** [comment]
yes this is an option because the [module that install EF](https://github.com/aws-samples/1click-hpc/blob/main/modules/10.install.enginframe.headnode.sh#L61)  backup an existing EF installation. So you are fine deleting an old cluster a creating a new one with the same name mounting the same FSx.

**Comment by rvencu** [comment]
on the post.install.sh, line 62
`export NICE_ROOT=$(jq --arg default "${SHARED_FS_DIR}/nice/${stack_name}" -r '.post_install.enginframe | if has("nice_root") then .nice_root else $default end' "${dna_json}")`

and on 10.install.enginframe.headnode.sh line 60
```
    if [[ -d "${SHARED_FS_DIR}/nice/${stack_name}" ]]; then
        mv  -f "${SHARED_FS_DIR}/nice/${stack_name}" "${SHARED_FS_DIR}/nice/${stack_name}.$(date "+%d-%m-%Y-%H-%M").BAK"
    fi
```

then multiple clusters can live side by side with the enginframe portals intact

**Comment by nicolaven** [comment]
yep! can you sent a PR specifically with this modification so I can incorporante
